tests/test_serialization.py

```
@@ -0,0 +1,102 @@
+        "difficulty_target": 1_000_000,
+        "nonce": 7,
+        "block_height": 3,
+    }
+    mutated = dict(base)
+    mutated["nonce"] = 8
+
+    assert serialize_block_header(base) != serialize_block_header(mutated)
+
+
+@pytest.mark.parametrize(
+    "payload,serializer,expected",
```

Ruff (0.15.1)

[warning] Wrong type passed to first argument of `pytest.mark.parametrize`; expected `tuple`

Use a `tuple` for the first argument

(PT006)